README.md

```diff
@@ -4,32 +4,27 @@ It features two processes that take turns calling functions `f()` and `g()` *in
 
 ![Screenshot](https://user-images.githubusercontent.com/5462697/41799127-e976f7a6-7678-11e8-8b06-fb6b4ee1d005.png)
 
-How?
+How it Works
 ===
-There’s nothing new in Lua being flexible: it allows to override all the memory management by passing a custom allocator to [`lua_newstate`](https://www.lua.org/manual/5.3/manual.html#lua_newstate).
+There’s nothing new in Lua being flexible. It allows to override all memory management by passing a custom allocator to [`lua_newstate`](https://www.lua.org/manual/5.3/manual.html#lua_newstate).
 The only things left are to create a shared memory mapping and implement an allocator working in it.
 
-This is more or less safe as long as Lua knows nothing about parallelism. Some things in the standard library do know, though; see the “Caveats” section.
```

Comment:
1. "more or less" does not really help the reader clarify how safe Lua is. It is best to avoid uncertainty by specifically mentioning the conditions in which Lua is not safe. Mentioning parallelism is one condition, and maybe more could have been added? I'll let you decide.

2. My aim was to remove the semi-colon and comma so "though" was removed as well. Maybe a better sentence would be: "See the 'Caveats' section for what things the standard library does know"?

3. I agree, "please" is not necessary in this sentence. Sorry about that. My 2nd point suggests a sentence without it.